app/controllers/posts_controller.rb

```diff
@@ -1,7 +1,36 @@
 class PostsController < ApplicationController
 
+  before_action :logged_in_user, only: [:new, :create]
+
+  def new
+    @post = current_user.posts.build if logged_in?
+  end
+
+  def create
+    @posts = current_user.posts.build(post_params)
+    if @posts.save
+      flash[:sucess] = "Post created"
+      redirect_to root_url
+    else
+      render 'static_pages/home'
+    end
+  end
+
   def index
     @posts = Post.all
   end
 
+  private
+
+  def post_params
+    params.require(:post).permit(:title, :content)
+  end
+
+  def logged_in_user
+    unless logged_in?
```

Comment:
Dans le lien que tu m'as donné : I also don’t mind using unless when you are just checking one condition and not using else like so… 

Mais je peux faire if !logged_in ?